baybe/surrogates/custom.py

```diff
@@ -96,7 +96,7 @@ def _estimate_moments(self, candidates: Tensor, /) -> tuple[Tensor, Tensor]:
             torch.from_numpy(results[1]).pow(2).to(DTypeFloatTorch),
         )
 
-    def _fit(self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor) -> None:
+    def _fit(self, train_x: Tensor, train_y: Tensor, context: Any) -> None:
```

Comment:
Was this change overlooked previously? Does not seem to be purely relateed to mypy issues